index.html

```diff
@@ -100,9 +100,9 @@ <h6 class="p" style="font-size: 30px;"><strong>Access Clipboard anywhere </stron
         </div>
     </div>
     <footer class="footer"
-        style="display: flex; justify-content: space-between; margin-top: 30px; width: 80%; text-align: center; margin:50px;">
-        <div class="copyright" style="width: 50px;">
-            <img src="/copyright.png" alt="copyright">
+        style="display: flex; gap: 150px; text-align: center; background: rgb(230, 224, 224); padding-left: 440px;">

```

Comment:
Hi @lillian-busingye ,

Kindly consider keeping the styling in the style.css sheet. It will make reviewing your code a whole lot easier. Having styles both in HTML and CSS makes the review tedious.